README.md

```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Install
+* Add `rg_instructor_analytics` to installed app 
```

Comment:
s/to installed/to the installed/ Please add the path to the settings file and move subsection down to the other subsection which relates settings file.

Reply:
I think - incorrect to add some file for adding this because we can setup it in different places.